### PR TITLE
Feature/global styles

### PR DESCRIPTION
**Pull request by DercilioFontes**
Global Styles #10

### CODE REVIEW

File: styles/globals.css [hunk 1]
```diff
@@ -0,0 +1,332 @@
...
+  border: 1px solid;
+  line-height: 24px;
+}
+
+button.primary {
+  background-color: var(--primary-blue);
+}
+
+button.primary:hover {
+  background-color: #374399;
```
Comment (marceloglacial) [inline styles/globals.css:123]: Please use variables for the colors

File: styles/globals.css [hunk 2]
```diff
@@ -0,0 +1,332 @@
...
+}
+
+button.primary:active {
+  background-color: #002167;
```
Comment (marceloglacial) [inline styles/globals.css:127]: Please use variables for the colors

File: styles/globals.css [hunk 3]
```diff
@@ -0,0 +1,332 @@
...
+  }
+
+  button,
+  a {
+    font-size: 20px;
+  }
+}
+
+/* Small devices (portrait tablets and large phones, 600px and up) */
+/* @media only screen and (min-width: 600px) {
```
Comment (marceloglacial) [inline styles/globals.css:319]: If is not in use, you can remove the comments

File: styles/globals.css [hunk 4]
```diff
@@ -0,0 +1,332 @@
...
+
+h5 {
+  font-size: 28px;
+}
+
+h5.uppercase {
+  text-transform: uppercase;
+}
+
+.b1 {
```
Comment (marceloglacial) [inline styles/globals.css:82]: It's not sure what b1 means. please use clear class names.

File: styles/globals.css [hunk 5]
```diff
@@ -0,0 +1,332 @@
...
+h1,
+h2,
+h3,
+h4,
+h5 {
+  font-weight: 700;
+}
+
+.display {
+  font-size: 58px;
```
Comment (marceloglacial) [inline styles/globals.css:55]: Please use rem for font sizes https://www.sitepoint.com/understanding-and-using-rem-units-in-css/

File: styles/globals.css [hunk 6]
```diff
@@ -0,0 +1,332 @@
...
+
+button,
+a {
+  font-size: 24px;
+  font-weight: 700;
+}
+
+button {
+  color: var(--neutral-white);
+  padding: 8px 33.5px;
```
Comment (marceloglacial) [inline styles/globals.css:112]: Let's try to use variables for spacing / margins. Will be easier to modify / update.

File: styles/globals.css [hunk 7]
```diff
@@ -0,0 +1,332 @@
...
+
+button.primary:hover {
+  background-color: #374399;
+}
+
+button.primary:active {
+  background-color: #002167;
+}
+
+button.primary.outline {
```
Comment (marceloglacial) [inline styles/globals.css:130]: If you are creating styles for the buttons, you should create a button component and the css module for it as well

**Review by marceloglacial — CHANGES_REQUESTED** [review]

File: styles/globals.css (hunk 4 above)
Comment (DercilioFontes) [reply styles/globals.css:82]: They are in the style guide on Figma. I didn't like it also, but it was to match them there. I was thinking to use body1 and so on.

File: styles/globals.css (hunk 5 above)
Comment (DercilioFontes) [reply styles/globals.css:55]: The style guide has those values, with a min and a max. I set max to Desktop view and min to mobile view. But I'm not sure if that was the intention.

I’ll change them to rem.

File: styles/globals.css (hunk 7 above)
Comment (DercilioFontes) [reply styles/globals.css:130]: It is following the style guide. I thought it will be used for all buttons.

File: styles/globals.css (hunk 1 above)
Comment (DercilioFontes) [reply styles/globals.css]: Done

File: styles/globals.css (hunk 3 above)
Comment (DercilioFontes) [reply styles/globals.css]: Done

File: styles/globals.css (hunk 2 above)
Comment (DercilioFontes) [reply styles/globals.css]: Done

File: styles/globals.css (hunk 4 above)
Comment (DercilioFontes) [reply styles/globals.css]: Changed

File: styles/globals.css (hunk 5 above)
Comment (DercilioFontes) [reply styles/globals.css]: Done

File: styles/globals.css (hunk 6 above)
Comment (DercilioFontes) [reply styles/globals.css]: Done

File: styles/globals.css (hunk 7 above)
Comment (DercilioFontes) [reply styles/globals.css:154]: I don't know that yet. It is a general styling based on the style guide
Screenshot 2023-02-27 at 10 25 19 PM

Comment (marceloglacial) [reply styles/globals.css:154]: Gotcha. let's keep it here and move it to a module file when we create the button component. 
Great job :)

**Review by marceloglacial — APPROVED** [review]

**Pull request merged**